tls/ttls.c

```diff
@@ -2101,31 +2088,47 @@ ttls_conf_version(TlsCfg *conf, int min_minor, int max_minor)
 	conf->max_minor_ver = max_minor;
 }
 
-#if defined(TTLS_SESSION_TICKETS)
-void ttls_conf_session_tickets_cb(TlsCfg *conf,
-				  ttls_ticket_write_t *f_ticket_write,
-				  ttls_ticket_parse_t *f_ticket_parse,
-				  void *p_ticket)
+/**
+ * Configure TLS session ticket callbacks (server only).
+ *
+ * @f_ticket_write callback should generate an encrypted and authenticated
+ * ticket for the session and write it to the output buffer. Here, ticket
+ * means the opaque ticket part of the NewSessionTicket structure of RFC 5077.
+ *
+ * @f_ticket_parse callback should parse a session ticket as generated by the
+ * corresponding ttls_ticket_write_t function, and, if the ticket is authentic
+ * and valid, load the session.
+ *
+ * The implementation is allowed to modify the first len bytes of the input
+ * buffer, eg to use it as a temporary area for the decrypted ticket contents.
+ *
+ * On server, session tickets are enabled by providing non-NULL callbacks.
+ */
+void
+ttls_conf_session_tickets_cb(TlsCfg *conf,
```

Comment:
Inconsistency: this code guarded by TTLS_SESSION_TICKETS is left intact, while at other places code is deleted.

Reply:
Yeah, the function must be deleted in #1054 (added a comment).